portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/NavTab.path

```diff
@@ -22,7 +22,7 @@
 </tr>
 <tr>
 	<td>TAB_LINK</td>
-	<td>//*[contains(@class,'nav-link') and normalize-space()='${key_tab}'][not(contains(@id,'ProductMenu') or contains(@href,'site_administration'))]</td>
+	<td>//button[contains(@class,'nav-link') and normalize-space()='${key_tab}'][not(contains(@id,'ProductMenu') or contains(@href,'site_administration'))]</td>
```

Comment:
This kind of change is like replacing "from 6 to 6.0", * means to use any field, and replace it with a button you will break other test cases and create a dependence with the field that needs to be a button.
so please, back to investigate it ...